geometry_msgs/PoseArray waypoints
---
bool success